### DISCUSSION TITLE
Flood UI Support?

### DISCUSSION THREAD

**Discussion by digitalbarrito** [haugene/docker-transmission-openvpn#2482] [General]
Curious to know if [Flood](https://github.com/jesec/flood) will be getting added as an alternative web UI option? I've seen discussions about it [here](https://github.com/haugene/docker-transmission-openvpn/issues/1661#issuecomment-792027824) and [here](https://github.com/haugene/docker-transmission-openvpn/pull/1663#issuecomment-780911645) and was curious if it's something that's been added, or is going to be, as there's a few things Flood has that Flood-for-transmission doesn't that would be nice to have.

**Comment by ilike2burnthing** [comment]
It was added a year ago - https://github.com/haugene/docker-transmission-openvpn/search?q=flood&type=commits

**Reply by digitalbarrito** [reply]
if I'm not mistaken that's [Food-for-transmission](https://github.com/johman10/flood-for-transmission) and not [Flood](https://github.com/jesec/flood)

Flood-for-transmission is a fork of Flood that was made when Flood didn't support Transmission, it's also not quite the same and lacks a few features Flood has.

**Reply by ilike2burnthing** [reply]
Ah, my bad, that's what I get for skimming your OP.
